Source/Protobuf/Failure.cs

```diff
@@ -36,18 +36,37 @@ public Failure(FailureId id, FailureReason reason)
         /// Implicitly convert <see cref="Failure" /> to <see cref="Contracts.Failure" />.
         /// </summary>
         /// <param name="failure"><see cref="Failure" /> to convert.</param>
-        public static implicit operator Contracts.Failure(Failure failure) =>
-            failure != null ?
-                new Contracts.Failure { Id = failure.Id.ToProtobuf(), Reason = failure.Reason }
-                : null;
+        public static implicit operator Contracts.Failure(Failure failure)
+        {
+            if (failure == null)
+            {
+                return null;
+            }
```

Comment:
See previous comments on similar suggestion
```suggestion
            if (failure == null) return null;
```

Reply:
Done